### PR TITLE
[UXE-4301] fix: show the message about deleting a payment method correctly

### PR DESCRIPTION
**Pull request by peterpaulo-azion**
### Explain what was fixed and the correct behavior.
* showing the response API message

### Does this PR introduce UI changes? Add a video or screenshots here.
image

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] bug: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [ ] Application responsiveness was tested to different screen sizes
- [ ] New tests are added to prevent the same issue from happening again
- [ ] UI changes are validated by a team designer
- [x] Code is formatted and linted
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [x] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

File: src/views/Billing/TabsView.vue [hunk 1]
```diff
@@ -151,6 +151,11 @@
         <TabPanel
           header="Bills"
           :disabled="accountBlocked"
+          :pt="{
+            headerAction: {
+              'data-testid': 'billing__bills-tab'
```
Comment (william-tome) [inline src/views/Billing/TabsView.vue:156]: Ajustar nomenclatura conforme documentação
`<location>__<element-name>__<element-type>`

File: src/services/billing-services/delete-payment-service.js [hunk 2]
```diff
@@ -21,6 +21,7 @@ const parseHttpResponse = (httpResponse) => {
     case 200:
       return 'Payment Method successfully deleted!'
     case 400:
+      if (httpResponse.body?.detail) throw new Error(httpResponse.body.detail).message
```
Comment (william-tome) [inline src/services/billing-services/delete-payment-service.js:24]: O teste unitário cobre essa ramificação no fluxo?

**Review by william-tome — COMMENTED** [review]
Por favor, coloca o link da execução manual do teste e2e do billing, pois ele não roda no fluxo principal

File: src/services/billing-services/delete-payment-service.js (hunk 2 above)
Comment (peterpaulo-azion) [reply src/services/billing-services/delete-payment-service.js:24]: Yes sir!

File: src/views/Billing/TabsView.vue (hunk 1 above)
Comment (peterpaulo-azion) [reply src/views/Billing/TabsView.vue:156]: Done!

**Review by william-tome — APPROVED** [review]

**Comment by peterpaulo-azion** [comment]
https://github.com/aziontech/azion-console-kit/actions/runs/10115932692

Workflow de testes

**Pull request merged**